src/components/IGVBrowser.js

```diff
@@ -75,8 +77,9 @@ const IGVBrowser = _.flow(
     }
 
     igvSetup()
-  })
 
+    return () => igvLibrary.current.removeAllBrowsers()
```

Comment:
Since `igvLibrary.current = igv` is set after an async import, it's probably worth checking if `igvLibrary.current` is set before calling `removeAllBrowsers`.

Reply:
I wonder if this would be a good use case for the `useGetter()` hook.

Reply:
`useGetter` stores a state variable in a ref so that a function can capture the ref and, through that ref, have access to the latest value of the state variable. Without the ref, the function would capture the value of the state variable at the time when the function was created. Since `igvLibrary` is a ref, and thus stable across renders, there's no need to wrap it in `useGetter`.

Reply:
Ah right; that makes sense. Thanks for the clarification!